packages/documentation-site/patternfly-docs/content/PatternFly-AI/rapid-prototyping/rapid-prototyping.md

```diff
@@ -0,0 +1,61 @@
+---
+id: Rapid prototyping
+section: PatternFly-AI
+source: about-rapid-prototyping
+sortValue: 2
+---
+
+# Rapid prototyping with AI-assisted PatternFly development
+
+This guide explores a workflow that enables teams to rapidly prototype PatternFly UIs directly in code using AI-powered development tools.
+
+## What is "vibe coding"?
+
+**Vibe coding** is an approach to AI-assisted development where you communicate the general "vibe" or feeling of what you want to create, rather than writing detailed specifications. Instead of providing precise technical requirements, you describe the user experience, visual aesthetic, or functional behavior you're aiming for, and let AI tools translate that vision into working PatternFly code.
+
+## Why rapid prototyping with PatternFly?
+
+Rapid prototyping with PatternFly code offers several key benefits:
+
+- **Early consistency**: Integrate PatternFly into prototyping processes as early as possible to foster design system consistency
+- **Enhanced collaboration**: Bridge the gap between product management, design, and engineering by speaking the same design language from the start
+- **Effective iteration**: Set up teams to collectively iterate efficiently by working directly with code
+- **Designer empowerment**: Equip teams with tools to rapidly build functional prototypes using PatternFly as a foundation
+
+## Getting started
+
+Choose your path based on your current situation:
+
+- **[Starting a new prototype](/patternfly-ai/rapid-prototyping/new-prototypes)** - Use the patternfly-react-seed application for optimal setup
+- **[Enhancing existing projects](/patternfly-ai/rapid-prototyping/enhancing-existing-projects)** - Integrate AI tools into your current codebase
+
+## Best practices for AI-assisted prototyping
+
+### Effective prompting with vibe coding
+
+When working with AI tools using the vibe coding approach:
+- Describe the user's emotional journey and what they need to accomplish
+- Focus on the experience rather than specific component names: "I want users to feel confident as they complete this critical task"
+- Use sensory and experiential language: "This should feel quick and lightweight" or "Users need to easily scan and compare options"
+- Be specific about PatternFly components you want to use when you know them, but don't hesitate to describe desired functionality when you don't
+- Reference existing PatternFly patterns when possible: "Similar to how the data list pattern works, but for this specific use case"
```

Comment:
```suggestion
- Describe the user's emotional journey and what they need to accomplish.
- Focus first on the experience, rather than specific component names. For example, "I want users to feel confident as they complete this critical task."
- Use sensory and experiential language. For example, "This should feel quick and lightweight." or "Users need to easily scan and compare options."
- If you do know of a specific PatternFly component you want used, be specific in your prompt.
- Reference existing PatternFly patterns when possible. For example, "Similar to how the data list pattern works, but for this specific use case."
```